src/input/pointer.js

```diff
@@ -354,6 +354,17 @@
                             }
                         }
                         break;
+
+                    case MOUSE_WHEEL:
+                        // event inside of bounds: trigger the MOUSE_WHEEL callback
+                        if (eventInBounds) {
+                            // trigger the corresponding callback
+                            if (triggerEvent(handlers, e.type, e, e.pointerId)) {
+                                handled = true;
+                                break;
+                            }
+                        }
+                        break;
```

Comment:
This looks identical to `POINTER_DOWN`. You should be able to add `case MOUSE_WHEEL:` above for a smaller patch.